source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift

```diff
@@ -12,7 +12,9 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol {
     private (set) var stackViewTopConstraint: NSLayoutConstraint?
     private (set) var stackViewBottomConstraint: NSLayoutConstraint?
     let hostConfig: ACSHostConfig
+    public var backgroundView: NSView?
```

Comment:
Do we really need this property?

Reply:
added for testcases to check for bleed property, haven't added testcases yet